gini/gini-vision-lib-android
# Feature default app dialog

When GVL is launched with an external file (aka "open with") we now check whether the app is set as default for that file type. If the app is the default, then we show an alert informing the user about this and telling them how to revert this. The user can either continue or be directed to the app's screen in the Android Settings app.

For images the dialog is shown in the review screens and for pdfs in the analysis screen.

*Note:* Showing an alert was only possible in the camera screen before. I had to touch a lot of files to allow the other screen to show dialogs, too, with minimal code duplication.

### How to test
Open a pdf or an image with one of the example apps and use the `ALWAYS` option. The dialog should be shown.
